### PR TITLE
prep to release 16.4

### PR DESCRIPTION
**Pull request by NadineWadi-Gallagher**

### CODE REVIEW

File: mobileconnectsdk/Mobile Connect SDK 16.4 Release Notes.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Both features require Command Centre server version 9.00 or higher.
+
+The SDK update also contains small bugfixes and BLE improvements.
+
+### Salto Audit Event Trail:
+This feature is automatically enabled once the Command Centre Server is setup with Salto and mobile event trail configuration is enabled for Cardholders.
+There is no flag to enable/disable this feature in the Mobile Connect SDK.
+
+### Aperio BLE Access:
+This feature requires Command Centre Server to be configured with BLE Aperio as well as the Aperio app per door.
```
Comment (bensheridanggl) [inline mobileconnectsdk/Mobile Connect SDK 16.4 Release Notes.md:17]: _This feature requires the Command Centre Server to be configured with BLE Aperio._

There are configuration steps required, but can we assume the site has already done that part? Or should we list all of the requirements here? Firmware upgrades, Command Centre configuration, and Aperio reader configuration with the PAP tool.

**Review by bensheridanggl — CHANGES_REQUESTED** [review]
_" as well as the Aperio app per door."_ This part isn't clear, I think we can drop it. Configuration requirements are covered in the User Guides.

File: mobileconnectsdk/Mobile Connect SDK 16.4 Release Notes.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Mobile Connect SDK 16.4 Release Notes
+
+Version 16.4 of the Mobile Connect SDK adds support for two main features:
+
+- Receive real-time Salto events for offline Salto readers on remote locations and transmit them to Command Centre and Salto servers via the Gallagher cloud without the need for badging at Salto update points.
```
Comment (adriclloydggl) [inline mobileconnectsdk/Mobile Connect SDK 16.4 Release Notes.md:5]: I guess we took this from somewhere but why 'real-time'? The events are historical at the point they're being uploaded to the cloud.

Comment (adriclloydggl) [inline mobileconnectsdk/Mobile Connect SDK 16.4 Release Notes.md:5]: Capitalise Cloud in 'Gallagher cloud' maybe

File: mobileconnectsdk/Mobile Connect SDK 16.4 Release Notes.md (hunk 1 above)
Comment (adriclloydggl) [reply mobileconnectsdk/Mobile Connect SDK 16.4 Release Notes.md:17]: +1 on the wording suggestion 

Seems reasonable without that extra info, this is primarily an overview

File: mobileconnectsdk/Mobile Connect SDK 16.4 Release Notes.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+### Salto Audit Event Trail:
+This feature is automatically enabled once the Command Centre Server is setup with Salto and mobile event trail configuration is enabled for Cardholders.
+There is no flag to enable/disable this feature in the Mobile Connect SDK.
+
+### Aperio BLE Access:
+This feature requires Command Centre Server to be configured with BLE Aperio as well as the Aperio app per door.
+The Mobile Connect SDK provides a feature flag to enable/disable Aperio BLE access.
+
+
+### Add:
```
Comment (adriclloydggl) [inline mobileconnectsdk/Mobile Connect SDK 16.4 Release Notes.md:21]: Added:

File: mobileconnectsdk/Mobile Connect SDK 16.4 Release Notes.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+A new value for `SdkFeature` is added to enable the Aperio integration feature.
```
Comment (adriclloydggl) [inline mobileconnectsdk/Mobile Connect SDK 16.4 Release Notes.md:22]: 'has been added'

**Review by adriclloydggl — CHANGES_REQUESTED** [review]

**Review by bensheridanggl — APPROVED** [review]

**Review by adriclloydggl — APPROVED** [review]

**Pull request merged**